.github/actions/e2e/slack/notify/action.yaml

```diff
@@ -20,8 +23,10 @@ runs:
     - id: get-run-name
       shell: bash
       run: |
-        if [[ ${{ github.event_name }} == "schedule" ]]; then
+        if [[ ${{ github.event_name }} == "schedule" && inputs.suite != "soak" ]]; then
           RUN_NAME="${{ inputs.suite }}-periodic"
+        elif [[ ${{ github.event_name }} == "schedule" && inputs.suite == "soak" ]]; then
+          RUN_NAME="$soak-periodic(${{ inputs.cluster_name }})"
```

Comment:
I wonder how much value there is in putting the cluster name in the channel since we can find the run url to track-down the failure if we see that this has failed in Slack